flask/src/main.py

```diff
@@ -242,6 +295,7 @@ def api():
 
 
 @app.route('/organization', methods=['GET'])
+@cache.cached(timeout=1000, key_prefix="flask.cache.organization")
```

Comment:
just checking this is intentional and not just leftover from testing different ways

Reply:
yup, thought i'd be nice to show more then a single operation in the cache view

Reply:
✅ confirmed this is in another transaction so we're not overloading flagship with too many cache spans